README.md

```diff
@@ -13,9 +13,44 @@ Build and install the provider so that Terraform can use it:
 make install
 ```
 
+## Test
+
+### Unit tests
+
+Unit tests run the provider against a `localhost_apiserver` that serves data read from `/synthetics/test-data.json`
+
+This allows to:
+- avoid the necessity of providing valid API credentials
+- avoid creating resources on remote server
+- make the test results more reliable
+
+
+To run unit tests:
+```bash
+make test
+```
+
+This will:
+1. Build and run `localhost_apiserver` that is a stub for kentik apiv6 server
```

Comment:
```suggestion
1. Build and run `test-api-server` that is a fake for Kentik API v6 server
```
I could be over the line here, but "fake" is more precise, because the server has internal state and responses aren't "stubbed". If one updates the internal state, server returns other responses.

Reply:
I'd formulate it like this:

1. Build and run `test-api-server` which emulates Kentik API v6 by returning static preconfigured responses

Reply:
Addressed: f1232530a413c31d588d1b567d9b1b5e007746be